scripts/s3BucketOptions/setBucketLifecyclePolicy.py

```diff
@@ -14,15 +14,15 @@ def main(
     destinationbucketaccess: str,
     destinationbucketsecret: str,
     destinationendpointurl: str,
-    noncurrentVersionExpirationDays: int,
+    noncurrentversionexpirationdays: int,
```

Comment:
These variable names are really difficult to read as a single word: use ``snake_case`` for variables and ``CamelCase`` for classes. I would start considering re-formatting all the python code in this repo as in ospar-simcore.

Same for the filenames.

Reply:
good point thanks for the input. feel free to open a ticket about it :) code formatting is imho out of the scope of this minor PR :)